CriteoPublisherSdk/Sources/Util/NSObject+Criteo.m

```diff
@@ -54,7 +54,6 @@ + (void)load {
 #pragma ide diagnostic ignored "InfiniteRecursion"
 - (void)cr_loadAd {
```

Comment:
This method seems to do nothing now and is part of the hack above that could be removed.

That hack above was swizzling the MPInterstitialAdController#loadAd method to automatically remove the Criteo bids keywords before.
This is part of MoPub support which can be removed now.

I guess the whole `load` override could simply be removed too then.